### PR TITLE
#69 Improved administration for Sponsor

### PR DESCRIPTION
**Pull request by Redpoint1**
FYI i don't like Django's solution for ManyToOne field. Eg. sponsored speakers (Sponsor model). Instead of ManyToMany & ForeignKey field solution (edit in popup, list of already existing speakers) you editing all fields of the related object INLINE. You cant add already existing speakers just new one.

### CODE REVIEW

File: konfera/admin.py [hunk 1]
```diff
@@ -70,10 +70,43 @@ class TalkAdmin(admin.ModelAdmin):
...
+        SponsoredSpeakersInline,
+    ]
+
+
+
 admin.site.register(Receipt)
 admin.site.register(Order)
 admin.site.register(Location)
-admin.site.register(Sponsor)
+admin.site.register(Sponsor, SponsorAdmin)
```
Comment (ZoltanOnody) [inline konfera/admin.py]: Please move the Admin registration just under the `SponsorAdmin` class. Same as `EventAdmin`.

File: konfera/admin.py [hunk 2]
```diff
@@ -70,10 +70,43 @@ class TalkAdmin(admin.ModelAdmin):
...
+    model = Speaker
+    verbose_name = _('Sponsored speaker')
+    verbose_name_plural = _('Sponsored speakers')
+    extra = 1
+
+
+class SponsorAdmin(admin.ModelAdmin):
+    list_display = ('title', 'type', 'url',)
+    list_filter = ('type',)
+    search_fields = ('=title',)
```
Comment (ZoltanOnody) [inline konfera/admin.py]: What does the `=` means in front of `title` ?

Comment (Redpoint1) [inline konfera/admin.py]: is a sign for case insensitive searching

Comment (ZoltanOnody) [inline konfera/admin.py]: thank you :)

File: konfera/admin.py (hunk 1 above)
Comment (Redpoint1) [inline konfera/admin.py]: Done

**Pull request merged**